simln-lib/src/lib.rs

```diff
@@ -173,6 +173,17 @@ pub type Amount = ValueOrRange<u64>;
 /// The interval of seconds between payments. Either a value or a range.
 pub type Interval = ValueOrRange<u16>;
 
+/// Payment activity options for running the simulation.
+pub enum ActivityOpts {
+    /// Defined payment activity provided by user that will be executed in the simulation.
+    DefinedActivity(Vec<ActivityDefinition>),
+    /// Generate random activity. Can optionally exclude nodes provided in ExcludeList.
+    Random(Option<ExcludeList>),
+}
+
+/// Nodes to exclude from sending/receiving in random activity.
+pub type ExcludeList = Vec<PublicKey>;
```

Comment:
I realize this is written up in the original issue, but wondering whether we want to have this `ExcludeList` in the library? Could be simpler to just have this enum and require that random provides a list of pubkeys that should be included.

The caller can just do that filtering by itself and pass the list through to the simulator. WDYT?

Reply:
I like the alias, was more wondering about whether an include/exclude list is a good API.

>  I'd prefer an optional exclusion list rather than inclusion.

👍